src/agora.js

```diff
@@ -0,0 +1,81 @@
+const AgoraRTC = require('agora-rtc-sdk')
+
+const startVideo = function(){
+let handleError = function(err){
+    console.log("Error: ", err);
+    };
+// Query the container to which the remote stream belong.
+let remoteContainer = document.getElementById("remote-container");
+
+// Add video streams to the container.
+function addVideoStream(elementId){
+    // Creates a new div for every stream
+    let streamDiv = document.createElement("div");
+    // Assigns the elementId to the div.
+    streamDiv.id = elementId;
+    // Takes care of the lateral inversion
+    streamDiv.style.transform = "rotateY(180deg)";
+    // Adds the div to the container.
+    remoteContainer.appendChild(streamDiv);
+};
+
+// Remove the video stream from the container.
+function removeVideoStream(elementId) {
+    let remoteDiv = document.getElementById(elementId);
+    if (remoteDiv) remoteDiv.parentNode.removeChild(remoteDiv);
+};
+
+
+let client = AgoraRTC.createClient({
+    mode: "rtc",
+    codec: "vp8",
+});
+
+client.init("bb8fa8a838674e38abd095b569e17cfc");
```

Comment:
Are these keys for the webcam? If so these should be stored in a different file and put in a git_ignore.